seahorn/trusty/user/base/lib/libc-trusty/sea_handle_table.c

```diff
@@ -212,6 +215,15 @@ handle_t sea_ht_new_port(bool secure) {
   return INVALID_IPC_HANDLE;
 }
 
+handle_t sea_ht_math_port(const char *path) {
+#define CASE(X)                                                                \
+  if ((strcmp(PHANDLE(X, path), path) == 0) && PHANDLE(X, active))             \
```

Comment:
this is too expensive. we can't have this in the model. no `memcmp` and definitely no `strcmp`.

If you need paths, restrict them to be one character length